### PR TITLE
chore: replace part of linter from golint to revive, fixed warning wh…

### PR DESCRIPTION
**Pull request by ishishow**
When I did a make lint on this repository, I got the following warnings.

スクリーンショット 2021-11-14 14 52 28

One is a note in golfingci-lint that golint is deperecated and revive is recommended
The other is a warning about the test code by staticcheck, one of the linters.

Therefore, I fixed it.

reference https://github.com/golangci/golangci-lint/issues/1892

Signed-off-by: ishishow <ishikawa.sho.infratop@gmail.com>

### CODE REVIEW

File: parser/json/json_test.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func TestJSONParser(t *testing.T) {
 	}
 
 	inputMap := input.(map[string]interface{})
-	if len(inputMap) < 0 {
+	if len(inputMap) <= 0 {
```
Comment (jalseth) [inline parser/json/json_test.go:39]: I think this should actually just be `==`, not `<=`.

**Review by jalseth — CHANGES_REQUESTED** [review]
Thank you for catching this and for the PR!

File: parser/json/json_test.go (hunk 1 above)
Comment (ishishow) [reply parser/json/json_test.go]: Oh, sure... I changed it! Thank you!

```len()``` did not return negative values, so I changed it, but I did not check it enough.
I found other similar parts and changed them.

**Comment by jpreese** [comment]
LGTM, thanks for your contribution! 🎉

**Review by jpreese — APPROVED** [review]

**Pull request merged**